packages/hash/api/src/graph/system-org.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  SYSTEM_ACCOUNT_NAME,
+  SYSTEM_ACCOUNT_SHORTNAME,
+} from "@hashintel/hash-shared/environment";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { types } from "@hashintel/hash-shared/types";
+import { getEntities } from "@hashintel/hash-subgraph/src/stdlib/element/entity";
+import {
+  extractEntityUuidFromEntityId,
+  Subgraph,
+  SubgraphRootTypes,
+} from "@hashintel/hash-subgraph";
+import { extractBaseUri } from "@blockprotocol/type-system";
+import { OrgModel, OrgSize } from "../model";
+import { PageDefinition, seedPages } from "../seed-data/seed-pages";
+
+// eslint-disable-next-line import/no-mutable-exports
+export let systemOrgAccountId: string;
+
+export const ensureSystemOrgAccountIdExists = async (params: {
+  graphApi: GraphApi;
+  logger: Logger;
+}) => {
+  const { graphApi, logger } = params;
+  const { data: existingOrgEntitiesSubgraph } =
+    await graphApi.getEntitiesByQuery({
+      filter: {
+        all: [
+          { equal: [{ path: ["version"] }, { parameter: "latest" }] },
+          {
+            equal: [
+              { path: ["type", "versionedUri"] },
+              { parameter: types.entityType.org.entityTypeId },
+            ],
+          },
+        ],
+      },
```

Comment:
We can't use the model classes here, because they rely on the system types being initialised (which happens after this function is called).